docs/source/tools/export/output-root/Export.daml

```diff
@@ -0,0 +1,89 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this file the golden master? Does the comparison somehow skip comments? Should we add this file to the license checking exclusion list?

Reply:
Answering my own question: yes, the comparison skips comments. To make sure we don't change the golden master every year, it would probably make sense to remove the license header.

Reply:
I think we update the headers in all our docs every year so including the golden master in that seems probably more consistent :shrug:

Reply:
No strong opinion on my side. I'm happy to keep it this way for consistency.